docs/faq.rst

```diff
@@ -297,23 +297,19 @@ directory specified prior to a ``launch --local-entry`` command
 
 Why should I migrate my existing workflow to use GitHub Apps and a .dockstore.yml?
 ----------------------------------------------------------------------------------
-Version 1.9.0 has added support for fetching workflows using GitHub Apps. Installing our GitHub App
+Release 1.9.0 of Dockstore has added support for syncing workflows using GitHub Apps. Installing our GitHub App
 onto your repository or organization will let Dockstore react whenever code is pushed to GitHub, ensuring that content
-on Dockstore is always automatically synced with GitHub. This requires the addition of a ``/.dockstore.yml`` file to
-your repository. This file contains version information such as workflow path, test parameter paths, etc that
-Dockstore will use to setup the version on Dockstore. If Dockstore sees this file in a repository with GitHub App installed,
-it can automatically update the corresponding workflows on Dockstore.
+on Dockstore is always automatically synced with GitHub. This means less work for publishers of content on Dockstore,
+and less waiting for content to update. This requires the addition of a ``/.dockstore.yml`` file to your repository.
+This file contains version information such as workflow path, test parameter paths, etc that Dockstore will use to setup
+the version on Dockstore. If Dockstore sees this file in a repository with GitHub App installed, it can automatically
+update the corresponding workflows on Dockstore. You can read more about it at :doc:`Dockstore GitHub Apps <./getting-started/github-apps>`.
 
 How does this change your development flow? Once you have a ``/.dockstore.yml`` file in your master/develop branches on GitHub, it should be
 there anytime you create a new branch. This means that Dockstore will pull in any new branch. If you want to edit version
 information, such as workflow path, you will have to update the ``/.dockstore.yml`` file directly on GitHub. You can no longer do this directly on Dockstore.
```

Comment:
That is correct though isn't it? Since you have to go to GitHub to edit the .dockstore.yml. Or do you mean it implies they must use the website and not the Git command line?